app/uk/gov/hmrc/excisemovementcontrolsystemapi/models/ExciseMovementResponse.scala

```diff
@@ -17,12 +17,14 @@
 package uk.gov.hmrc.excisemovementcontrolsystemapi.models
 
 import play.api.libs.json.{Json, OFormat}
+import uk.gov.hmrc.excisemovementcontrolsystemapi.repository.model.MovementId
 
 case class ExciseMovementResponse(
-    status: String,
-    localReferenceNumber: String,
-    consignorId: String,
-    consigneeId: Option[String] = None
+                                   status: String,
+                                   movementId: MovementId,
```

Comment:
With the MovementId being a class, when this comes back in the response will it be a nested structure such as:

``` json
{
  "status": "Accepted",
  "movementId": {
    "value": "some-uuid-string"
  },
  "consignorId": "Consignor",
  "consigneeId": "Consignee",
 "localReferenceNumber": "LRN"
}
```
If so, would it be better to just store it as a String and pull the value out of the movementId?

Reply:
We're going to chnage this in the next PR, we've just discovered it not working now :)

Reply:
Could this be done for this PR as it relates to these changes and then we can close this comment here and move onto the next bit of work?